components/table/src/table.spec.ts

```diff
@@ -827,3 +857,39 @@ class TestIndicatorApp {
   color: 'error' | 'warning' = 'error';
   active = true;
 }
+
+@Component({
+  selector: 'dt-test-app',
+  template: `
+    <dt-table [dataSource]="dataSource">
+      <ng-container dtColumnDef="host" dtColumnAlign="text">
+        <dt-header-cell *dtHeaderCellDef>Host</dt-header-cell>
+        <dt-cell *dtCellDef="let row">{{ row.host }}</dt-cell>
+      </ng-container>
+
+      <dt-header-row *dtHeaderRowDef="['host']"></dt-header-row>
+      <dt-row *dtRowDef="let row; columns: ['host']"></dt-row>
+      <custom-empty-state dtEmptyState></custom-empty-state>
```

Comment:
VSCode complains at this line that it can't find `custom-empty-state`. I'm not sure how to fix that, but everything seems to work...